docs/readthedocs/io/csvfiles.md

```diff
@@ -24,80 +24,180 @@ You may extend / alter the naming with pre- or suffix by calling `new EntityPers
 
 ### Input
 
-| Model                             | File Name                                                                                 |
-|:----------------------------------|:------------------------------------------------------------------------------------------|
-| operator                          | *prefix_* operator_input *_suffix*                                                        |
-| node                              | *prefix_* node_input *_suffix*                                                            |
-| line                              | *prefix_* line_input *_suffix* <br> *prefix_* line_type_input *_suffix*                   |
-| switch                            | *prefix_* switch_input *_suffix*                                                          |
-| two winding transformer           | *prefix_* transformer2w_input *_suffix* <br> *prefix_* transformer2w_type_input *_suffix* |
-| three winding transformer         | *prefix_* transformer3w_input *_suffix* <br> *prefix_* transformer3w_type_input *_suffix* |
-| measurement unit                  | *prefix_* measurement_unit_input *_suffix*                                                |
-| biomass plant                     | *prefix_* bm_input *_suffix* <br> *prefix_* bm_type_input *_suffix*                       |
-| combined heat and power plant     | *prefix_* chp_input *_suffix* <br> *prefix_* chp_type_input *_suffix*                     |
-| electric vehicle                  | *prefix_* ev_input *_suffix* <br> *prefix_* ev_type_input *_suffix*                       |
-| electric vehicle charging station | *prefix_* evcs_input *_suffix*                                                            |
-| fixed feed in facility            | *prefix_* fixed_feed_in_input *_suffix*                                                   |
-| heat pump                         | *prefix_* hp_input *_suffix* <br> *prefix_* hp_type_input *_suffix*                       |
-| load                              | *prefix_* load_input *_suffix*                                                            |
-| photovoltaic power plant          | *prefix_* pc_input *_suffix*                                                              |
-| electrical energy storage         | *prefix_* storage_input *_suffix* <br> *prefix_* storage_type_input *_suffix*             |
-| wind energy converter             | *prefix_* wec_input *_suffix* <br> *prefix_* wec_type_input *_suffix*                     |
-| schematic node graphic            | *prefix_* node_graphic_input *_suffix*                                                    |
-| schematic line graphic            | *prefix_* line_graphic_input *_suffix*                                                    |
+```{eval-rst}
+.. list-table::
+   :widths: auto
+   :header-rows: 0
```

Comment:
If one compares tables from before and after, it becomes obvious that we had the first row designated as a header row (results in bold font). Maybe you could try that for all tables in this file.

Reply:
Sorry, should I take a look at #985 again?

Reply:
I think it's enough to just set it in this file for now.